tools/topology/topology2/include/platform/intel/dmic-generic.conf

```diff
@@ -0,0 +1,196 @@
+<include/dais/dmic.conf>
+<include/dais/pdm_config.conf>
+# This file should be included from other top-level topology files. So do not include other
+# conf's in this file.
+
+#
+# Arguments to be passed when compiling this conf file. All of them come with default values.
+# These can be modified by passing the argument values with alsatplg as follows:
+# alsatplg -D DMIC0_ID=4,DMIC1_ID=5,DMIC_CH_COUNT=4 -p -c dmic_generic.conf -o output.conf
+#
+
+Define {
+	# Link ID for DMIC 480KHz in the machine driver
+	DMIC0_ID		6
+	DMIC0_PIPELINE_ID	10
+	DMIC0_PCM_ID		6
+	DMIC0_EQIIR_COEFF	"eqiir_highpass_40hz_20db_48khz"
+
+	# Link ID for DMIC 16Khz in the machine driver
+	DMIC1_ID		7
+	DMIC1_PIPELINE_ID	11
+	DMIC1_PCM_ID		7
+	DMIC1_EQIIR_COEFF	"eqiir_highpass_40hz_20db_16khz"
+
+	# Number of active PDM controllers.
+	NUM_PDM		1
+
+	# Source widget name in the pipeline that connects to the DMIC0 DAI
+	DMIC0_PIPELINE_SOURCE	"buffer.10.3"
+
+	# Source widget name in the pipeline that connects to the DMIC1 DAI
+	DMIC1_PIPELINE_SOURCE	"buffer.11.3"
+
+	# Stream name for the host widget for the DMIC0 pipeline. This should be the same as the
+	# PCM capabilities for the PCM that this widget is connected to
+	DMIC0_HOST_STREAM_NAME	"Highpass Capture 6"
+
+	# Stream name for the host widget for the DMIC0 pipeline. This should be the same as the
+	# PCM capabilities for the PCM that this widget is connected to
+	DMIC1_HOST_STREAM_NAME	"Highpass Capture 7"
+
+	# DMIC0 EQIIR byte control name
+	DMIC0_EQIIR_BYTES_NAME "eqiir_bytes_10"
```

Comment:
Would it be possible to replace hard coded 10 with $DMIC0_PIPELINE_ID or endpoint name? It's rather difficult to find manually the control if the suffix number is different than what pipeline # suggests?